pkg/importer/transport.go

```diff
@@ -36,15 +34,11 @@ import (
 )
 
 const (
-	cancelTimeout           = 300 * time.Second
-	dockerTarLayerMediaType = "application/vnd.docker.image.rootfs.diff.tar"
-	ociTarLayerMediaType    = "application/vnd.oci.image.layer.v1.tar"
-	whFilePrefix            = ".wh."
+	whFilePrefix = ".wh."
 )
 
 func commandTimeoutContext() (context.Context, context.CancelFunc) {
-	ctx := context.Background()
-	return context.WithTimeout(ctx, cancelTimeout)
+	return context.WithCancel(context.Background())
```

Comment:
@awels I think we cannot say what a "generic" timeout for any situation should be. So I got rid of it; the importer will now patiently download images for as long as it takes.